torch/distributed/optim/utils.py

```diff
@@ -24,6 +24,18 @@
     optim.Adamax: _FunctionalAdamax,
 }
 
+
+# Interface to insert a new functional optimizer to functional_optim_map
+# Example::
+#     >>> import the new functional optimizer
+#     >>> from xyz import fn_optimizer
+#     >>> from torch.distributed.optim.utils import register_functional_optim
+#     >>> register_functional_optim(key, fn_optimizer)
```

Comment:
`key` is not defined in this example

Reply:
@rohan-varma updated with a sample dummy registration test to show case. let me know your views